verify/README

```diff
@@ -2,77 +2,91 @@
 
 This is the deployment container and configuration for the Cockpit
 integration tests. This documentation is for deployment on Fedora 22+
-or RHEL 7+
+or RHEL 7+.
 
-This is for a single operating system. Choose the operating system to
-test ahead of time.
+Use the following commands to run the verify container as a one off:
 
-## How to deploy
+   $ sudo yum -y install docker atomic
+   $ sudo systemctl start docker
+   $ sudo atomic run cockpit/infra-verify
```

Comment:
Is it a requirement to start these through `atomic`? What's the main difference to starting them through `docker` directly? Supposedly the difference for installation is that `atomic install` creates a systemd service for you, but for one-off?